src/components/NavBar.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './navbar.css';
+
+const NavBar = () => (
+  <div className="header">
+    <h1 className="nav-title">Math Magicians</h1>
+    <ul className="nav-list">
+      <li>
+        <Link to="/" className="active-link">
+          <p>Home</p>
+        </Link>
+      </li>
+      <li>
+        <Link to="/calc" className="active-link">
+          <p>Calculator</p>
+        </Link>
+      </li>
+      <li>
+        <Link to="/quote" className="active-link">
+          <p>Quote</p>
+        </Link>
+      </li>
+    </ul>
+  </div>
```

Comment:
[OPTIONAL] I think it's important that we take care of the HTML semantics, even if we're working on React. This means using adequate tags for elements like `header` to wrap the header of a page and `nav` to wrap elements use to navigate the app.

Reply:
I took care of it, so thanks a lot for the feedback and for the review!!